test/NullPointer.java

```diff
@@ -17,6 +17,12 @@ private static void throwAndCatch(Object o) {
 
   public static void main(String[] args) {
     try {
+      ((Object) null).getClass();
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+
+    try {
```

Comment:
To be honest, I don't know.  There must be some other mechanism by which a class can get a valid vtable besides the VM boot process, but I don't know what/where it is offhand.